crates/floresta-chain/src/pruned_utreexo/error.rs

```diff
@@ -44,6 +46,10 @@ pub enum BlockValidationErrors {
 impl Display for BlockValidationErrors {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         match self {
+            BlockValidationErrors::InvalidCoinbase(e) => {
+                write!(f, "Invalid coinbase: {:?}", e)
+            },
+            BlockValidationErrors::TooManyCoins => write!(f, "Moving more coins that exists"),
```

Comment:
Is inevitable the creation of more error types. Hope that's Ok.

Reply:
It's `ok`. I would say it's preferable...